Kernel/makeall.sh

```diff
@@ -7,9 +7,21 @@ cd "$script_path"
 # Get user and group details for setting qemu disk image ownership
 build_user=$(id -u)
 build_group=$(id -g)
+fast_mode=
 export build_user
 export build_group
 
+while [ "$1" != "" ]; do
+    case $1 in
+        -f | --fast )           fast_mode=1
+                                ;;
+        -h | --help )           echo "-f or --fast: build fast without cleaning or running tests"
+                                exit 1
```

Comment:
Should this not be using 0 as the return code?

Reply:
Oops, yes.